.npmignore

```diff
@@ -1,5 +1,10 @@
 node_modules
 coverage
 *.log
-.codeclimate
```

Comment:
.codeclimate needs to stay. It contains a secret key for publishing test coverage etc to code climate. Ideally this should be done by CI, but until then we need to leave it in.

Reply:
@cressie176 I see, solved!